projects/m2/022-anagrams/js/index.js

```diff
@@ -0,0 +1,23 @@
+const string1 = "evil"
+const string2 = "live"
+const string3 = "hello"
+const string4 = "world"
+
+function anagrams(s1,s2){
+    const count = [];
+    for(i=0; i<s1.length;i++){
+        count.push(s1[i])
+        
+    }
+    for(n=0;n<s2.length;n++){
+        if (count.includes(s2[n])){
```

Comment:
What happens if as `s2` I pass `"lllllliiiiiiivvvvvveeeeeeeee"`? ;-)

`const string2 = "lllllliiiiiiivvvvvveeeeeeeee";`

Is the solution correct? Fix it. ;-)